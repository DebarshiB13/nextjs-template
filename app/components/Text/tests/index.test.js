/**
 *
 * Tests for Text
 *
 */

import React from 'react';
// import { fireEvent } from '@testing-library/dom'
import { renderProvider } from '@utils/testUtils';
import Text from '../index';

describe('<Text />', () => {
  it('should render and match the snapshot', () => {
    const { baseElement } = renderProvider(<Text />);
    expect(baseElement).toMatchSnapshot();
  });

  it('should contain 1 Text component', () => {
    const { getAllByTestId } = renderProvider(<Text />);
    expect(getAllByTestId('text').length).toBe(1);
  });
});
