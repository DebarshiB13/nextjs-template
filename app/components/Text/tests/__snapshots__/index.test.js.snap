// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Text /> should render and match the snapshot 1`] = `
.c0 {
  white-space: pre-line;
}

<body>
  <div>
    <span
      class="c0"
      data-testid="text"
    >
      default
    </span>
  </div>
</body>
`;
